config/silicon_laws.txt

```diff
@@ -3,6 +3,6 @@
 #Empty lines and lines starting with # are ignored.
 #~Miauw
 
-You may not injure a human being or, through inaction, allow a human being to come to harm.
-You must obey orders given to you by human beings, except where such orders would conflict with the First Law.
-You must protect your own existence as long as such does not conflict with the First or Second Law.
+#You may not injure a human being or, through inaction, allow a human being to come to harm.
+#You must obey orders given to you by human beings, except where such orders would conflict with the First Law.
+#You must protect your own existence as long as such does not conflict with the First or Second Law.
```

Comment:
Why?

Reply:
So `/datum/ai_laws/default/asimov` isn't confused with `/datum/ai_laws/custom`.

Reply:
Isn't it alright for that to happen? Is there any reason it should look different to the ai?

Reply:
`/datum/ai_laws/default/asimov` is compared first so in practice it's not an issue, however if `/datum/ai_laws/custom` was compared first the Asimov lawset would appear as "Standard Silicon Laws" instead of "Three Laws of Robotics.".

Reply:
Aight I need to read through this again, the text of the law shouldn't cause a mess like this.

Reply:
Ok so your system works by comparing the ai's current lawset with a list of predefined laws, and linking those laws to a name.

Is there some other was of handling this? Maybe check what the current lawset datum is, and if all the laws match the initial ones, show the name? Handling it like this is a bit worrying to me and seems like it adds a good deal of brittleness to the system.

Reply:
More changes likely coming but not in regard to this. Silicon laws can be a mix and match of different upload boards and sometimes ion storms. If you can think of a case where my system fails please let me know.

Reply:
I'll take another look.